src/components/formData.json

```diff
@@ -0,0 +1,91 @@
+{
+        "title": "Sign up form",
+        "fields": [
+                {
+                        "key": "first_name",
+                        "label": "First Name",
+                        "isRequired": true,
+                        "type": "text",
+                        "placeholder": "Your First Name"
+                },
+                {
+                        "key": "last_name",
+                        "label": "Last Name",
+                        "isRequired": true,
+                        "type": "text",
+                        "placeholder": "Your Last Name"
+                },
+                {
+                        "key": "dob",
+                        "label": "Date of Birth",
+                        "isRequired": true,
+                        "type": "date"
+                },
+                {
+                        "key": "phone_number",
+                        "label": "Phone Number",
+                        "isRequired": true,
+                        "type": "phone",
+                        "placeholder": "Your Phone Number"
+                },
+                {
+                        "key": "region",
+                        "label": "Region",
+                        "isRequired": true,
+                        "type": "dropdown",
+                        "placeholder":"Region",
+                        "helpText":"Region in which your school belongs to",
+                        "options":["Bhopal", "Chandigarh", "Hyderabad", "Pune", "Shillong"]
+                },
+                {
+                        "key": "state",
+                        "label": "State",
+                        "isRequired": true,
+                        "type": "dropdown",
+                        "placeholder":"State",
+                        "dependant":true,
+                        "dependantField": "region",
+                        "dependantFieldMapping":{
+                                "Bhopal": ["Chhattisgarh", "Madhya Pradesh", "Odisha"],
+                                "Lucknow": ["Uttarakhand", "Uttar Pradesh"],
+                                "Chandigarh": [
+                                  "Chandigarh",
+                                  "Himachal Pradesh",
+                                  "Jammu & Kashmir",
+                                  "Ladakh",
+                                  "Punjab"
```

Comment:
do we need to put this in every form that uses state info in JNV? what about state -> jnv district mapping? can't we use some fixed json for these mappings